content/neuromorphic-computing/research/guide/badge-system/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "ONR Badge & Certificate System"
+description: "An explanation of the 'ONM Community Approved' badge, what it signifies, and how approved projects are showcased on a dedicated certificate page."
+showTableOfContents: true
+type: "research"
+---
+
+## The "ONM Community Approved" Badge
+
+The "ONM Community Approved" badge is a mark of quality and trust within the neuromorphic ecosystem. It signifies that a project, paper, or resource has successfully passed the Open Neuromorphic Research (ONR) community peer-review process.
+
+Earning the badge means your project will be featured in our [Approved Research Registry](/neuromorphic-computing/research/papers/), with a dedicated page that serves as a permanent certificate of its approval.
+
+### What the Badge Represents
+
+When you see this badge, it means the work has been evaluated against our [official review criteria](/neuromorphic-computing/research/guide/review-criteria/) and has been recognized for its:
+
+-   **Clarity:** The work is well-documented and easy to understand.
+-   **Reproducibility:** The results can be independently verified by the community.
+-   **Impact:** The contribution is valuable and relevant to the neuromorphic field.
+-   **Openness:** The work adheres to open-source principles with permissive licensing.
+
+## The Certificate Page
+
+Each project that successfully passes the ONR review is granted a permanent "certificate" page within our [Approved Research Registry](/neuromorphic-computing/research/papers/). This page serves as the official record of the community's endorsement and typically includes:
```

Comment:
Should it be `/papers/`? What if people submit datasets/libraries/...? What about `/research/certificates`?

Feel free to ignore if this is nitpicking.

Reply:
It could have been certificates, I considered various options but felt that 'neuromorphic research papers' was a juicier SEO target. :)